### PR TITLE
Fix missing Expense ID in virtual card notification email

### PR DESCRIPTION
**Pull request by SudharakaP**
Related to the freshdesk ticket; https://opencollective.freshdesk.com/a/tickets/137283. Here we are not passing the expense and thus ends up with a missing expense id.

### CODE REVIEW

**Comment by znarf** [comment]
Seems it was broken in https://github.com/opencollective/opencollective-api/pull/7815.

How could that have been prevented?

**Review by znarf — APPROVED** [review]

**Comment by SudharakaP** [comment]
> Seems it was broken in #7815.
> 
> How could that have been prevented?

It wasn't supported in the first place. Could have prevented if I've added a test to check this link as well (in addition to checking the body). I've just done this.

File: server/paymentProviders/stripe/virtual-cards.js [hunk 1]
```diff
@@ -216,6 +216,7 @@ export const processAuthorization = async (stripeAuthorization, stripeEvent) =>
         VirtualCardId: virtualCard.id,
         responsibleAdmin: responsibleAdmin.activity,
         collective: collective.activity,
+        expense,
```
Comment (Betree) [inline server/paymentProviders/stripe/virtual-cards.js:219]: I'm surprised that we're using `expense`, not `expense.info here. Aren't we storing a huge payload in there? @SudharakaP could you post an example of `activiy.data.expense` as generated by this?

Comment (SudharakaP) [reply server/paymentProviders/stripe/virtual-cards.js:219]: Yes, this is my mistake. I should have used `expense.info` here. I've corrected that in the PR; 

The paypload for expense is,

```
Expense {
  dataValues: {
    createdAt: 2022-11-03T15:50:57.659Z,
    updatedAt: 2022-11-03T15:50:57.659Z,
    legacyPayoutMethod: 'manual',
    feesPayer: 'COLLECTIVE',
    id: 1,
    UserId: 5,
    CollectiveId: 7,
    FromCollectiveId: 5,
    currency: 'USD',
    amount: 100,
    description: 'Virtual Card charge: stripe',
    VirtualCardId: 'card_BM7mGwp1Ea8RtL',
    lastEditedById: 5,
    status: 'PROCESSING',
    type: 'CHARGE',
    incurredAt: 2022-11-03T15:50:57.650Z,
    data: { authorizationId: 'stripe_auth_id' },
    HostCollectiveId: null,
    payeeLocation: null,
    longDescription: null,
    PayoutMethodId: null,
    privateMessage: null,
    invoiceInfo: null,
    vat: null,
    deletedAt: null,
    tags: null,
    RecurringExpenseId: null
  },
  _previousDataValues: {
    UserId: 5,
    CollectiveId: 7,
    FromCollectiveId: 5,
    currency: 'USD',
    amount: 100,
    description: 'Virtual Card charge: stripe',
    VirtualCardId: 'card_BM7mGwp1Ea8RtL',
    lastEditedById: 5,
    status: 'PROCESSING',
    type: 'CHARGE',
    incurredAt: 2022-11-03T15:50:57.650Z,
    data: { authorizationId: 'stripe_auth_id' },
    id: 1,
    HostCollectiveId: null,
    payeeLocation: null,
    longDescription: null,
    legacyPayoutMethod: 'manual',
    PayoutMethodId: null,
    privateMessage: null,
    invoiceInfo: null,
    vat: null,
    feesPayer: 'COLLECTIVE',
    createdAt: 2022-11-03T15:50:57.659Z,
    updatedAt: 2022-11-03T15:50:57.659Z,
    deletedAt: null,
    tags: null,
    RecurringExpenseId: null
  },
  uniqno: 1,
  _changed: Set(0) {},
  _options: {
    isNewRecord: true,
    _schema: null,
    _schemaDelimiter: '',
    attributes: undefined,
    include: undefined,
    raw: undefined,
    silent: undefined
  },
  isNewRecord: false
}
```

whereas for `expense.info` it's,
```
{
  type: 'DEBIT',
  id: 1,
  UserId: 5,
  CollectiveId: 7,
  FromCollectiveId: 5,
  HostCollectiveId: null,
  currency: 'USD',
  amount: 100,
  description: 'Virtual Card charge: stripe',
  category: undefined,
  tags: null,
  legacyPayoutMethod: 'manual',
  vat: null,
  privateMessage: null,
  lastEditedById: 5,
  status: 'PROCESSING',
  incurredAt: 2022-11-03T15:52:16.080Z,
  createdAt: 2022-11-03T15:52:16.089Z,
  updatedAt: 2022-11-03T15:52:16.089Z
}
```

**Pull request merged**